sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_shared/async_client_base.py

```diff
@@ -87,7 +86,7 @@ def __init__(
         if api_version is None:
             api_version = KeyVaultClient.DEFAULT_API_VERSION
 
-        config = config or self.create_config(credential, api_version=api_version, **kwargs)
+        config = self.create_config(credential, api_version=api_version, **kwargs)
```

Comment:
>config = self.create_config(credential, api_version=api_version, **kwargs) [](start = 8, length = 74)

Should create_config be marked as private now? _create_config?

Reply:
sounds good